forge-game/src/main/java/forge/game/ability/effects/ManaEffect.java

```diff
@@ -106,25 +111,26 @@ public void resolve(SpellAbility sa) {
                             choiceString.append(" ");
                         }
                         choiceString.append(choice);
+                        choiceSymbols.append(MagicColor.toSymbol(choice));
```

Comment:
Is there a way to choose Colorless from a Snow source?

Reply:
Are you asking if there's a card/combo that lets you make that choice? The notification only shows up when generating mana with a nonmana ability, which really narrows the scope on top of being Snow, and most mana choosing effects are "of any color" which narrows it further. Maybe put Elemental Resonance on Ulalek, then Rimefeather Owl puts an ice counter on Elemental Resonance? Doesn't look like there's a notification in that branch though...

Reply:
Cool. If someone finds a weird corner case we can display Snow if applicable. Looks good.